### PR TITLE
Rev codestyle version to 3.9

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -30,7 +30,7 @@
     <RoslynDiagnosticsNugetPackageVersion>3.3.3-beta1.21105.3</RoslynDiagnosticsNugetPackageVersion>
     <MicrosoftCodeAnalysisNetAnalyzersVersion>6.0.0-preview1.21054.10</MicrosoftCodeAnalysisNetAnalyzersVersion>
     <MicrosoftCodeAnalysisTestingVersion>1.0.1-beta1.20623.3</MicrosoftCodeAnalysisTestingVersion>
-    <CodeStyleAnalyzerVersion>3.8.0</CodeStyleAnalyzerVersion>
+    <CodeStyleAnalyzerVersion>3.9.0</CodeStyleAnalyzerVersion>
```
Comment (mavasani) [inline eng/Versions.props:33]: Interesting, I would have expected at least some new violations on command line with adopting a new CodeStyle package. Is CodeStyle enforcement broken on build? I’ll play around with it today.

Comment (CyrusNajmabadi) [reply eng/Versions.props:33]: No clue :)  i took it to mean that our code is just that awesome.

**Review by jmarolf — APPROVED** [review]

**Comment by CyrusNajmabadi** [comment]
@mavasani should i merge and we can check things later?  or do you want me to hold off?

**Comment by mavasani** [comment]
> @mavasani should i merge and we can check things later? or do you want me to hold off?

Feel free to merge.

**Pull request merged**